webapp/javascript/components/FlameGraph/FlameGraphComponent/Header.module.css

```
@@ -10,6 +10,11 @@
   width: 100%;
 }
 
+.flamegraph-header button {
```

```suggestion
.flamegraphHeader button {
```

No personal preference, just out of consistency https://github.com/css-modules/css-modules#naming

---

done!